crates/bevy_ecs/src/system/query.rs

```diff
@@ -975,6 +975,35 @@ where
         self.state
             .is_empty(self.world, self.last_change_tick, self.change_tick)
     }
+
+    /// Returns `true` if the given [`Entity`] matches the query.
```

Comment:
I don't propose that you take any action on this comment, just noting down my thoughts.

By a strict reading of these docs, a valid implementation of this function would be `{true}`; we don't specify the behaviour if the condition is false. It is however intended to be read as `if and only if`.

However, this fits the pattern of documentation in `std`, e.g. [`Vec::contains`](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.contains). I tried to find a [rust-lang/rust](https://github.com/rust-lang/rust) issue on these doc items, but failed to do so.